### DISCUSSION TITLE
Option to match the slider color to the light color

### DISCUSSION THREAD

**Discussion by ironcrafter54** [nicufarmache/lovelace-big-slider-card#5] [Ideas]
I came across this card a few days ago and it's great. One thing that I think would make this card even cooler if you had an option to match the slider color to the color of the light, like the mushroom cards.

**Comment by nicufarmache** [comment]
I'll try to add this in the next version

**Reply by ironcrafter54** [reply]
Cool! this will make dashboards using this card even better.

**Comment by nicufarmache** [comment]
I added the new `colorize` option in v1.0.8 you can try it now
```yaml
colorize: true
```

**Reply by ironcrafter54** [reply]
Works like a charm, it just makes the slider much better. One small thing I noticed was during the transition when the slider changes color the slider is a box rather than beveled to match the bevel of the theme. Not really something that's a problem, just thought I would point it out.

https://user-images.githubusercontent.com/99451924/222938831-bf1c9bff-d93c-4401-bc2a-abf6cae660a9.mov

**Reply by nicufarmache** [reply]
Thanks for letting me know about this.
I found the source of the problem and I'll fix it in the next version.

**Reply by nicufarmache** [reply]
You should update to the latest version and see if the problem is fixed now.
